spec/support/shared_contexts/inherited_proofing.rb

```diff
@@ -0,0 +1,33 @@
+RSpec.shared_context 'inherited_proofing' do
+  let(:auth_code) {}
+  let(:private_key) { private_key_from_store_or(file_name: 'va_ip.key') }
+  let(:payload) { { inherited_proofing_auth: auth_code, exp: 1.day.from_now.to_i } }
+  let(:jwt_token) { JWT.encode(payload, private_key, 'RS256') }
+  let(:request_uri) {
+    "#{InheritedProofing::Va::Service::BASE_URI}/inherited_proofing/user_attributes"
+  }
+  let(:request_headers) { { Authorization: "Bearer #{jwt_token}" } }
+  let(:user_attributes) {
```

Comment:
this is a lot of pieces... I worry that a shared context makes it really hard to figure out where they come from in specs. What about a module with constants? ex `InheritedProofingSupport::USER_ATTRIBUTES` etc etc? that way it's clear where things come from?

Reply:
Sure, I can give that a try

Reply:
I prefer to use context when working with specs but I have broken them out into two files to better separate concerns and to provide more clarity. I've also added some additional comments to make it clearer how the files can be used